.github/workflows/release.yaml

```diff
@@ -0,0 +1,131 @@
+on:
+  push:
+    tags:
+      - 'litani-*'
+
+name: Create Release and package
+
+jobs:
+  get-version-information:
+    name: Get Version Information
+    runs-on: ubuntu-20.04
+    outputs:
+      version: ${{ steps.split-version.outputs._1 }}
+    steps:
+      - uses: jungwinter/split@v1
+        id: split
+        with:
+          msg: ${{ github.ref }}
+          seperator: '/'
+      - uses: jungwinter/split@v1
+        id: split-version
+        with:
+          msg: ${{ steps.split.outputs._2 }}
+          seperator: '-'
+  perform-release:
+    name: Perform Release
+    runs-on: ubuntu-20.04
+    needs: get-version-information
+    outputs:
+      version: ${{ needs.get-version-information.outputs.version }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          LITANI_VERSION: ${{ needs.get-version-information.outputs.version }}
+        with:
+          tag_name: litani-${{ env.LITANI_VERSION }}
+          release_name: litani-${{ env.LITANI_VERSION }}
+          body: |
```

Comment:
Is there any way we can include the changelog entry in here?

Reply:
As discussed in the meetings, we will be triggering this workflow only on tag push to release branch.